hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -184,6 +189,16 @@ public class DatanodeConfiguration extends ReconfigurableConfig {
   )
   private int blockDeleteQueueLimit = 5;
 
+  @Config(key = "block.delete.command.worker.interval",
+      type = ConfigType.LONG,
+      defaultValue = "2000",
+      tags = {DATANODE},
+      description = "The interval between DeleteCmdWorker execution of " +
+          "delete commands."
+  )
+  private long blockDeleteCommandWorkerInterval =
```

Comment:
We can make it more user-friendly and improve type safety by changing this to `ConfigType.TIME`, which accepts values with unit.  With that, default value can be changed to `2s` and variable type to `Duration`.

Reply:
Thanks @adoroszlai  for the comment and review.
I will update soon.